static/scripts/index.js

```diff
@@ -0,0 +1,513 @@
+                }
+                //Style first col if needed
+                if (col === 0 && row < lenFixedRowsText) {
+                    TD.style.fontWeight = 'bold';
+                    TD.style.textAlign = 'left';
+                }
+                //(row%2) === 1 to avoid empty lines
+                if (row >= lenFixedRowsText && (row % 2) === 0 && col < lenFixedHeaders) {
+                    const rowValue = this.getDataAtRow(row);
+                    if (col === 5 || col === 6) {
+                        const total_load = rowValue[4];
+                        const load_q1 = rowValue[5];
+                        const load_q2 = rowValue[6];
+
+                        //First case: load Q1 + Q2 = total load
+                        //Second case: load Q1 or Q2 = half of total load
+                        if (load_q1 + load_q2 > total_load) {
+                            TD.style.backgroundColor = 'red';
+                        } else if ((total_load === 4 || total_load === 2) && (total_load / value <= 2 || load_q1 + load_q2 === total_load)) {
+                            TD.style.backgroundColor = 'green';
+                        } else if (total_load === 1) {
+                            if (value === 1) {
+                                TD.style.backgroundColor = 'green';
+                            } else {
+                                if ((load_q1 === 1 && load_q2 === 0) || (load_q1 === 0 && load_q2 === 1)) {
+                                    TD.style.backgroundColor = 'green';
+                                }
+                            }
+                        }
+                    }
+                }
+            },
+            customBorders: [
+                {
+                    range: {
+                        from: {
+                            row: 0,
+                            col: 0,
+                        },
+                        to: {
+                            row: nbrLines,
+                            col: 6,
+                        },
+                    },
+                    end: {
+                        width: 2,
+                        color: 'black'
+                    },
+                },
+                {
+                    range: {
+                        from: {
+                            row: 0,
+                            col: 0,
+                        },
+                        to: {
+                            row: 5,
+                            col: nbrCols,
+                        },
+                    },
+                    bottom: {
+                        width: 2,
+                        color: 'black'
+                    },
+                },
+                {
+                    row: 5,
+                    col: 6,
+                    bottom: {
+                        width: 2,
+                        color: 'black'
+                    }
+                },
```

Comment:
You can probably compact this snippet a little bit. The expanded dict is harder to read than it would be in a more compacted form.